src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.spec.js

```diff
@@ -441,4 +450,16 @@ describe('The calInboxInvitationMessageBlueBarController', function() {
       expect(ctrl.isActionable()).to.equal(false);
     });
   });
+
+  describe('The translateStartDate function', function() {
+    it('should translate the format of start date in the local language', function() {
+      var ctrl = initCtrl('REQUEST', '1234', '1');
+      const expectedLocale = 'fr';
```

Comment:
Mixing `var` & `const` is not cool :wink:

Reply:
You're one day too late :trollface:.

Reply:
Yep. I know. Just an invest for the future :stuck_out_tongue_winking_eye:

Reply:
thanks :p